lib/GADS/Schema/Result/Report.pm

```diff
@@ -221,10 +221,10 @@ has data => (
         my $record_id = $self->record_id;
         $record->find_current_id($record_id);
 
-        while ( my $layout = $layouts->next ) {
-            my $column = $gads_layout->column( $layout->layout_id );
+         while ( my $layout = $layouts->next ) {
+            my $column = $gads_layout->column( $layout->layout_id, permission=>'read' ) or next;
```

Comment:
Thanks. Just fussy white space now - `while` has gained a space in front of it, and just personal preference but `permission => 'read'` keeps that consistent too :)